app/src/main/res/layout/main_fragment.xml

```diff
@@ -14,72 +14,67 @@
             android:layout_height="wrap_content"
 
             android:layout_marginTop="41dp"
-            android:hint="Select a recipe"
+            android:gravity="center_horizontal"
 
+            android:hint="@string/selectaRecipe"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:text="Select a Recipe"
-            tools:layout_editor_absoluteX="0dp"
-            android:gravity="center_horizontal" />
+            app:layout_constraintTop_toTopOf="parent" />
 
         <ListView
             android:id="@+id/lstRecipes"
-            android:layout_width="405dp"
-            android:layout_height="629dp"
-            android:layout_marginTop="86dp"
-            android:layout_marginEnd="6dp"
-            android:layout_marginRight="6dp"
-            android:layout_marginBottom="16dp"
-            app:layout_constraintTop_toBottomOf="@+id/rcpSearch"/>
-
-        <Button
-            android:id="@+id/btnShoppingList"
-            android:layout_width="100dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="10dp"
-            android:layout_marginLeft="10dp"
-            android:text="Shopping List"
-            android:textSize="9dp"
-            app:layout_constraintStart_toStartOf="parent"
-            tools:layout_editor_absoluteY="105dp" />
-
-        <Button
-            android:id="@+id/btnCatagories"
-            android:layout_width="100dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="40dp"
-            android:layout_marginLeft="40dp"
-            android:text="Catagories"
-            android:textSize="9dp"
-            app:layout_constraintStart_toEndOf="@+id/btnShoppingList"
-            tools:layout_editor_absoluteY="105dp" />
-
-        <Button
-            android:id="@+id/btnSchedule"
-            android:layout_width="100dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="40dp"
-            android:layout_marginLeft="40dp"
-            android:layout_marginEnd="10dp"
-            android:layout_marginRight="10dp"
-            android:text="Schedule"
-            android:textSize="9dp"
```

Comment:
Delete all the text button, I think it will be much better if we use the image button. Because images are easier to understand than words, which means more friendly to the users.